packages/wix-storybook-utils/src/LiveCodeExample/LiveCodeExample.tsx

```diff
@@ -35,6 +35,7 @@ export interface Props {
   previewWarning?({ onConfirm: Function }): React.ReactNode | null;
   hints?: ComponentsHints;
   forceInitialCode?: boolean;
+  playgroundWrapper?: Function;
```

Comment:
```suggestion
  playgroundWrapper?: StoryConfig['story']['playgroundWrapper'];
```

let's reuse the type from `StoryConfig`, you can get it from here:

```ts
import { StoryConfig } from '../typings/story-config';
```